src/utils/immutable.js

```diff
@@ -0,0 +1,9 @@
+import { List } from 'immutable';
+
+export const concatImmutableLists = (...args) =>
+  args.filter(list => list).reduce((acc, nextArray) => acc.concat(nextArray), List());
+
+export const immutableListToShallowJS = (list, defaultValue = []) =>
+  list ? list.toArray().map(p => p.toJSON()) : defaultValue;
+
+export const hasObjectTruthyValue = obj => !!(obj && !!obj.find(value => value));
```

Comment:
The name sounds odd to me. How about just using `hasTruthyValue`?

Reply:
renamed